test/unit/Site.test.js

```diff
@@ -351,6 +353,86 @@ test('Site read correct user defined variables', async () => {
   expect(subsub.number).toEqual('9999');
 });
 
+test('Site convert generates correct assets', async () => {
+  const json = {
+    'src/template/page.ejs': PAGE_EJS,
+
+    'inner/index.md': INDEX_MD_DEFAULT,
+
+    'inner/_Footer.md': '',
+    'inner/_Sidebar.md': '',
+    'inner/Home.md': '',
+  };
+  fs.vol.fromJSON(json, '');
+  await Site.initSite('inner/');
+  await new Site('inner/', 'inner/_site/').convert();
+
+  // number of files
+  const paths = Object.keys(fs.vol.toJSON());
+  const originalNumFiles = Object.keys(json).length;
+  const expectedNumBuilt = 15;
+  expect(paths.length).toEqual(originalNumFiles + expectedNumBuilt);
+
+  // footer
+  expect(fs.existsSync(path.resolve('inner/_markbind/layouts/default/footer.md'))).toEqual(true);
+
+  // site navigation
+  expect(fs.existsSync(path.resolve('inner/_markbind/layouts/default/navigation.md'))).toEqual(true);
```

Comment:
If possible, are you able to check the contents that are auto-generated here?

Reply:
Was this addressed else where?

If no I was thinking of doing a check on the content of the navigation file here. The file is generated by the conversion and the results should be predictable.

Reply:
> Was this addressed else where?

Sorry, I forgot to mention that I put the content check in the second unit test (in `Site.test.js#419`). Is that what you mean?